best-practices/audio-setup.md

```diff
@@ -0,0 +1,17 @@
+# Audio setup
+
+## Known issues and workarounds
+
+### My colleagues tell me that I sound like a pilot
+
+- Open the sound settings on your computer and check if your microphone volume is too high. If it's nearly maxed out, try lowering the volume a bit. Maxing out the microphone can causes severe quality degregation.
```

Comment:
This is just minor nitpicking but "degregation " should be "degradation". :-)

Reply:
I knew this was wrong, I just couldn't point out the exact problem :D